staging/src/k8s.io/apiserver/pkg/server/options/serving.go

```
@@ -172,6 +241,18 @@ func (s *ServingOptions) AddDeprecatedFlags(fs *pflag.FlagSet) {
 	fs.MarkDeprecated("port", "see --insecure-port instead.")
 }
 
+func (s *ServingOptions) ApplyTo(c *server.Config) error {
```

This error is not handled by the callers.